Marlin/Configuration.h

```diff
@@ -708,7 +708,7 @@
                                  // It is assumed that when logic high = filament available
                                  //                    when logic  low = filament ran out
 #if ENABLED(FILAMENT_RUNOUT_SENSOR)
-  const bool FIL_RUNOUT_INVERTING = false; // set to true to invert the logic of the sensor.
+  #define FIL_RUNOUT_INVERTING false // set to true to invert the logic of the sensor.
```

Comment:
Can you do the same in all the `Configuration.h` files in `example_configuration` too? Thanks!